### ISSUE TITLE
Cannot add FlSpot with same "y" in LineChart 

### ISSUE THREAD

**Issue by HoseinHaqiqian** [imaNNeoFighT/fl_chart#879]
Hi, i was going to add some Spot to the chart with same "y" and different "x".
but i ran into an error :
```
Unsupported operation: Result of truncating division is Infinity or NaN

#0      double._trunc_div (dart:core-patch/double.dart:56:57)
#1      double.~/ (dart:core-patch/double.dart:52:12)
#2      AxisChartPainter._drawGrid (package:fl_chart/src/chart/base/axis_chart/axis_chart_painter.dart:284:27)
#3      AxisChartPainter.paint (package:fl_chart/src/chart/base/axis_chart/axis_chart_painter.dart:40:5)
#4      LineChartPainter.paint (package:fl_chart/src/chart/line_chart/line_chart_painter.dart:79:11)
#5      RenderLineChart.paint (package:fl_chart/src/chart/line_chart/line_chart_renderer.dart:75:14)
#6      RenderObject._paintWithContext (package:flutter/src/rendering/object.dart:2409:7)
#7      PaintingContext.paintChild (package:flutter/src/rendering/object.dart:189:13)

```
i'm using **fl_chart : ^0.41.0**

**Comment by iambaofeng** [comment]
i have the same question,when i use data like x:['2021-08','2021-09','2121-10','2021-11','2021-12','2022-01'] y:[0.0,0.0,0.0,0.0,0.0,0.0,0.0]  it will catch a error ,like this=>
![image](https://user-images.githubusercontent.com/26156319/150253441-411c02c0-824a-4818-8a5f-89113192f65e.png)

**Comment by erlangparasu** [comment]
same problem

**Comment by erlangparasu** [comment]
how to solve: `maxY` must greater than 0 `minY`
so i calculate manually and return default value for maxY

```dart
    int maxValue = 0;
    for (var item in myListValue) {
      if (item > maxValue) {
        maxValue = a;
      }
    }

    if (maxValue == 0) {
      maxValue = 10;
    }

    return LineChartData(
      // lineTouchData: lineTouchData1,
      // gridData: gridData,
      // titlesData: titlesData1,
      // borderData: borderData,
      // lineBarsData: ...,
      // minX: ...,
      // maxX: ...,
      minY: 0.toDouble(),
      maxY: maxValue.toDouble(),
    );
```

**Comment by imaNNeoFighT** [comment]
As I explained in the issue template, Please provide me a reproducible code (a main.dart file).

**Comment by imaNNeoFighT** [comment]
Hi. 
We have just fixed it in [0.45.0](https://github.com/imaNNeoFighT/fl_chart/releases/tag/0.45.0).
Please check it out!
